src/itemEditableInfo/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+import { TextDisplayType } from 'text'
+import Item from '_utils/item'
+import { A11yProps, pickA11yProps } from '_utils/interfaces'
+import { color } from '_utils/branding'
+
+export interface ItemEditableInfoProps extends A11yProps {
+  readonly className?: Classcat.Class
+  // A label for a modifiable user value.
+  readonly label: string
+  // A modifiable user value.
+  readonly value: string
+  // A href to follow if the modifiable value is activated
+  readonly href?: string | JSX.Element
+  // Prevent modification of the user input.
+  // Used to trigger the behavior of the 'ItemEditableInfo non-editable' from the specs.
+  readonly disabled?: boolean
+  readonly ariaLabel?: string
+}
+
+const ItemEditableInfo = (props: ItemEditableInfoProps) => {
+  const { className, label, value, href = null, disabled = false } = props
+  const a11yAttrs = pickA11yProps<ItemEditableInfoProps>(props)
+
+  const extraProps = {
+    isClickable: true,
+    href,
+    // For the ItemEditableInfo, the value (entered previously by the user)
+    // is the most important info and is visually bigger than the
+    // label for it.
```

Comment:
Not possible to replace for Typography?

Reply:
I would love to but I will need to recreate a new Item implmentation.
We need a separate project and PRs for that.